gramps/gen/lib/tableobj.py

```diff
@@ -142,6 +143,15 @@ def get_handle(self):
         """
         return self.handle
 
+    def is_empty(self) -> bool:
+        """
+        Return true if the object is not associated with a database handle.
+
+        :returns: true if there is no database handle associated with the object, false otherwise
+        :rtype: bool
+        """
+        return self.handle
```

Comment:
If the return type is bool, should the return be:

```suggestion
        return self.handle is None
```

Reply:
I believe there's an implicit conversion to `bool`

I'll solve anyway by deleting this method. It's no longer required with the raise error approach.